openshift/helpers.sh

```diff
@@ -0,0 +1,47 @@
+
+function is_set_or_fail() {
```

Comment:
What's the purpose of this file here?

Reply:
I am using deploy.sh script to run the cronjob on devcluster, it is a helper script for `deploy.sh`